### PR TITLE
Added port to webhook-service, generated bundle with v1.0.1

### PR DESCRIPTION
**Pull request by jpkrohling**
Based on the discussion from the operator-framework mailing list, I forked your repository and ran `make bundle`, resulting in the diff from this PR.

Note that I'm using `operator-sdk` v1.0.1 and `controller-gen` v0.3.0.

Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: bundle/manifests/webhook-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -167,12 +179,8 @@ spec:
     url: https://your.domain
   version: 0.0.1
   webhookdefinitions:
-  - admissionReviewVersions:
-    - v1beta1
-    - v1
-    containerPort: 443
-    targetPort: 4343
-    deploymentName: webhook-operator-webhook
+  - admissionReviewVersions: null
```
Comment (jpkrohling) [inline bundle/manifests/webhook-operator.clusterserviceversion.yaml:182]: I guess this explains why I'm seeing an error when deploying my operator :-)

File: bundle/manifests/webhook-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -185,15 +193,11 @@ spec:
       - UPDATE
       resources:
       - webhooktests
-    sideEffects: None
+    sideEffects: null
```
Comment (jpkrohling) [inline bundle/manifests/webhook-operator.clusterserviceversion.yaml:196]: And once I manually fix the `admissionReviewVersions` in my bundle, I get a message about this field being invalid as well.

File: config/crd/patches/webhook_in_webhooktests.yaml [hunk 3]
```diff
@@ -16,3 +16,4 @@ spec:
         namespace: webhook-operator-system
         name: webhook-service
         path: /convert
+        port: 443
```
Comment (jpkrohling) [inline config/crd/patches/webhook_in_webhooktests.yaml:19]: Without this, I get a message like this:

```
WARN[0000] Warning: Value webhook.operators.coreos.io/v2, Kind=WebhookTest: provided API should have an example annotation 
ERRO[0000] Error: Field spec.conversion.webhookClientConfig.service.port, Value 0: spec.conversion.webhookClientConfig.service.port: Invalid value: 0: port is not valid: must be between 1 and 65535, inclusive 
make: *** [Makefile:115: bundle] Error 1
```

**Comment by jpkrohling** [comment]
I also just tried using `controller-gen` 0.2.5, which is what the `Makefile` in this branch seems to install when it's absent, but it didn't generate a bundle significantly different than this PR:

```diff
diff --git a/bundle/manifests/webhook.operators.coreos.io_webhooktests.yaml b/bundle/manifests/webhook.operators.coreos.io_webhooktests.yaml
index 77146f9..568c71b 100644
--- a/bundle/manifests/webhook.operators.coreos.io_webhooktests.yaml
+++ b/bundle/manifests/webhook.operators.coreos.io_webhooktests.yaml
@@ -3,7 +3,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     cert-manager.io/inject-ca-from: webhook-operator-system/webhook-operator-serving-cert
-    controller-gen.kubebuilder.io/version: v0.3.0
+    controller-gen.kubebuilder.io/version: v0.2.5
   creationTimestamp: null
   name: webhooktests.webhook.operators.coreos.io
 spec:
diff --git a/config/crd/bases/webhook.operators.coreos.io_webhooktests.yaml b/config/crd/bases/webhook.operators.coreos.io_webhooktests.yaml
index 1bb7b2e..6d69c2c 100644
--- a/config/crd/bases/webhook.operators.coreos.io_webhooktests.yaml
+++ b/config/crd/bases/webhook.operators.coreos.io_webhooktests.yaml
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.3.0
+    controller-gen.kubebuilder.io/version: v0.2.5
   creationTimestamp: null
   name: webhooktests.webhook.operators.coreos.io
 spec:
```

**Comment by jpkrohling** [comment]
@awgreene sorry for pinging you directly, but did you have a chance to take a look at this one?

File: bundle/manifests/webhook-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (awgreene) [reply bundle/manifests/webhook-operator.clusterserviceversion.yaml:182]: This [is fixed](https://github.com/operator-framework/operator-sdk/pull/3903) in version 1.0.1 of the operator SDK.

File: bundle/manifests/webhook-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (awgreene) [reply bundle/manifests/webhook-operator.clusterserviceversion.yaml:196]: This [is fixed](https://github.com/operator-framework/operator-sdk/pull/3903) in version 1.0.1 of the operator SDK.

File: config/crd/patches/webhook_in_webhooktests.yaml (hunk 3 above)
Comment (awgreene) [reply config/crd/patches/webhook_in_webhooktests.yaml:19]: I'll update the CRD to use a pointer to an int and default to 443.

Comment (awgreene) [reply config/crd/patches/webhook_in_webhooktests.yaml:19]: This is a bug that I'll look into - we should be defaulting to 443.

File: bundle/manifests/webhook-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (jpkrohling) [reply bundle/manifests/webhook-operator.clusterserviceversion.yaml:182]: This one seems to have missed the v1.0.1 train.

File: bundle/manifests/webhook-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (jpkrohling) [reply bundle/manifests/webhook-operator.clusterserviceversion.yaml:196]: This too :-)

File: bundle/manifests/webhook-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (jpkrohling) [reply bundle/manifests/webhook-operator.clusterserviceversion.yaml:182]: https://github.com/operator-framework/operator-sdk/commit/88a6f988c8d85ab686b802ad524dbd823f3227d9

Comment (awgreene) [reply bundle/manifests/webhook-operator.clusterserviceversion.yaml:182]: My mistake - the commit isn't in v1.0.1 of the SDK.

**Pull request closed without merging**